tests/test_regex.py

```diff
@@ -50,6 +50,30 @@ def test_dom_mutation_fail():
     assert _do_test_raw('foo.DOMAttrModified = bar;').failed()
 
 
+def test_bug_1200929():
+    """Test that XPIProvider and AddonManagerInternal are not used."""
+
+    real_world_usage = (
+        'XPIProviderBP = Cu.import("resource://gre/modules/XPIProvider.jsm", {});')
+
+    err = _do_test_raw(real_world_usage)
+    assert err.failed()
+    assert err.signing_summary['high'] == 1
+
+    err = _do_test_raw('// XPIProvider')
+    assert err.failed()
+    assert err.signing_summary['high'] == 1
+
+    err = _do_test_raw('AddonManagerInternal.getAddonByID(1234);')
+    assert err.failed()
+    assert err.signing_summary['high'] == 1
+
+    err = _do_test_raw('// AddonManagerInternal')
```

Comment:
Same here. If somebody gets back to this code in the future and sees, they might wonder why we wanted this specifically ;)